example-singleServer-basic.html

```diff
@@ -4,6 +4,20 @@
 <head>
     <title>LibreSpeed Example</title>
     <link rel="shortcut icon" href="favicon.ico">
+    <style type="text/css">
+        html,body{
+            border:none; padding:0; margin:0;
```

Comment:
```suggestion
            border: none;
            padding: 0;
            margin: 0;
```
Please fix formatting